flaml/tune/tune.py

```diff
@@ -374,15 +468,24 @@ def easy_objective(config):
         try:
             import optuna as _
 
-            SearchAlgorithm = BlendSearch
+            if lexico_objectives is None:
+                SearchAlgorithm = BlendSearch

```

Comment:
the try...except should be inside the "if" branch.

Reply:
resolve